lib/src/core/services/networking/dio_service.dart

```diff
@@ -0,0 +1,165 @@
+import 'package:dio/dio.dart';
```

Comment:
There are also a api_service_dio, which is the good one?

Reply:
api_service_dio implements api_interface, so methods fom dio_service are implemented there. But again, there are many ways of doing things and I am not sure which one is best